lib/services/ntpd.pm

```diff
@@ -18,6 +18,10 @@ use utils;
 use strict;
 use warnings;
 
+# default ntp service type and name
+my $service_type = 'Systemd';
```

Comment:
this is needed or not? the parameter should defined in function full_ntpd_check ?

Reply:
we have tests/console/ntp.pm called these function. like follow so we'd set default 
service_name and service_type.

sub run {
    select_console 'root-console';
    services::ntpd::install_service();
    services::ntpd::enable_service();

Reply:
understand, so we need put logic like：
if (version=11sp4) then {
     service_type=xx
} else {
     service_type=yy
}